pallets/processor-manager/src/lib.rs

```diff
@@ -129,10 +132,16 @@ pub mod pallet {
     pub(super) type ManagerCounter<T: Config> = StorageMap<_, Blake2_128, T::AccountId, T::Counter>;
 
     #[pallet::storage]
-    #[pallet::getter(fn processor_last_seen)]
     pub(super) type ProcessorHeartbeat<T: Config> = StorageMap<_, Blake2_128, T::AccountId, u128>;
 
+    #[pallet::storage]
+    #[pallet::getter(fn processor_last_seen)]
+    pub(super) type ProcessorLastSeen<T: Config> = StorageMap<_, Identity, T::AccountId, u128>;
```

Comment:
I do not get why we need to rename this storage map. Their names are equivalent in this context and there is no point in messing the storage with a migration.

Reply:
the idea was to iterate over all processors and filter the once to display for selection, but turns out iteration is not possible.
For now I removed the migration to a reversible key.